src/libs/actions/Policy.js

```diff
@@ -266,6 +296,38 @@ function isAdminOfFreePolicy(policies) {
         && policy.role === CONST.POLICY.ROLE.ADMIN);
 }
 
+/**
+ * Modify NetworkRequestQueue Onyx on invite member API call
+ * Return new pendingInvitations emails and remainingMembers that need API call to remove
+ *
+ * @param {String} policy
+ * @param {Array} members
+ * @returns {Array} [newPendingInvitations, remainingMembers]
+ */
+function removePendingInvitations(policy, members) {
+    // No pendingInvitations if network is online
+    if (!Network.isNetworkOffline()) {
+        return [policy.pendingInvitations, members];
+    }
```

Comment:
This early return suggests that this should never be part of this method. even if needed pass `isOffline` from param.

Reply:
All right, maybe I will move to upper function.